mr/src/itest/java/org/elasticsearch/hadoop/Provisioner.java

```diff
@@ -42,7 +42,7 @@
         // init ES-Hadoop JAR
         // expect the jar under build\libs
         try {
-            File folder = new File(".." + File.separator + "build" + File.separator + "libs" + File.separator).getCanonicalFile();
+            File folder = new File("build" + File.separator + "libs" + File.separator).getCanonicalFile();
```

Comment:
Let's avoid hardcoding "build" and instead use `project.buildDir`.

Reply:
I totally agree, I'll fix this in a follow up.